keyboards/model_f_labs/f62/keymaps/via/config.h

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2023 Matthew Wolf
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#undef BOOTMAGIC_ENABLE
+#undef BOOTMAGIC_LITE
```

Comment:
`BOOTMAGIC_ENABLE = no` would be the way to disable this, and in the rules.mk. 

Alternatively, you can suppy a different `bootmagic_lite` function to change or disable the functionality, if needed.

Reply:
From keyboards/model_f_labs/xwhatsit_capsense_matrix.c

#if CAPSENSE_CAL_ENABLED
#if defined(BOOTMAGIC_ENABLE) || defined(BOOTMAGIC_LITE)
#error "Calibration is not supported in conjunction with BOOTMAGIC, because calibration requires that no keys are pressed while the keyboard is plugged in"
#endif
#endif

CAPSENSE_CAL_ENABLED is required for the xwhatis controller in the keyboard.  

The contents of keyboards/model_f_labs/f62/keymaps/via/config.h is needed for all the key maps Added the contents to keyboards/model_f_labs/f62/wcass/config.h.